src/client/locale/I18.ts

```
@@ -0,0 +1,324 @@
+				case 'o':
+					return formatBaseX(number, 8, prefixBaseX, leftJustify, minimumWidth, precision, zeroPad, customPadChar);
+				case 'x':
+					return formatBaseX(number, 16, prefixBaseX, leftJustify, minimumWidth, precision, zeroPad, customPadChar);
+				case 'X':
+					return formatBaseX(number, 16, prefixBaseX, leftJustify, minimumWidth, precision, zeroPad, customPadChar).toUpperCase();
+				case 'u':
+					return formatBaseX(number, 10, prefixBaseX, leftJustify, minimumWidth, precision, zeroPad, customPadChar);
+				case 'i':
+				case 'd':
+					number = Math.round(number - number % 1);
+					const absValue = Math.abs(number).toString();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)